assets/src/edit-story/karma/fixture/containers/canvas.js

```diff
@@ -125,4 +125,8 @@ class Frame extends Container {
   constructor(node, path) {
     super(node, path);
   }
+
+  get textContent() {
+    return this.node.querySelector('[data-testid="textFrame"]')?.innerHTML;
```

Comment:
I feel like `querySelector('p')` is the best here and fairly semantic.

Reply:
Yeah, makes sense. It's the `<TextFrame />` so I guess we can consider the DOM fairly static as it's not actually displayed.

Reply:
Fixed in 59e09b67d.